lib/medzoo/ResNet3D_VAE.py

```diff
@@ -235,7 +235,7 @@ def forward(self, x):
         x = x.view(-1, self.linear_in_dim)
         x = self.linear_1(x)
         mu = x[:, :self.split_dim]
-        logvar = torch.log(x[:, self.split_dim:])
+        logvar = x[:, self.split_dim:]
```

Comment:
That's correct. The `log` here is misplaced because it is indirectly assumed in the latent vector `x`. I summon @black0017 for a second opinion

Reply:
I see! At the moment of writing this I was not aware of the fact that `log` is redundant. Thanks!!!